main.go

```diff
@@ -89,6 +108,7 @@ func (m *HarborCli) PullRequest(ctx context.Context, directoryArg *dagger.Direct
 	log.Println("Pull-Request tasks completed successfully 🎉")
 }
 
+// Function that responsible for handling tasks related to releasing the Harbor CLI tool.
```

Comment:
```suggestion
// Release handles building, tagging, and publishing the Harbor CLI tool.
```

update description to be concise clear.